whipper/program/cdparanoia.py

```diff
@@ -196,7 +196,7 @@ def getTrackQuality(self):
 
         # don't go over a 100%; we know cdparanoia reads each frame at least
         # twice
-        return min(frames * 2.0 / reads, 1.0)
+        return min(frames * 2.0 / reads, 1.0) if reads else 0
```

Comment:
This doesn't seem very Pythonic to me. Something like
```python
if reads:
    return min(…)
else:
    return 0
```
Or maybe even just assigning to a variable:
```python
quality = 0
if reads:
    quality = min(frames * 2.0 / reads, 1.0)
return quality
```